ReleaseNotes.md

```diff
@@ -11,6 +11,14 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+
+### 05/12/2021 -
+##### Version microsoft/oms:ciprod00512021 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod00512021 (linux)
```

Comment:
the agent version doesnt look right

Reply:
oops, good catch. Looks like Ganga fixed that.